client/lib/wp/sync-handler/cache-index.js

```diff
@@ -65,12 +65,16 @@ export const cacheIndex = {
 	 * @param {String} key - record key
 	 * @return {Promise} promise
 	 */
-	addItem( key ) {
+	addItem( key, pageSeriesKey = null ) {
```

Comment:
I'm a bit concerned about this key into the record. It seems a very specific case related with pagination. What would happen if you have to handle other cases in a near future.
Anyway it's something that we can modify/refact later.

Also can you update @jsdoc here?